diff --git a/tests/pass/unix/test07.asm b/tests/pass/unix/test07.asm
index 60ad737..f6ac470 100644
--- a/tests/pass/unix/test07.asm
+++ b/tests/pass/unix/test07.asm
@@ -4,67 +4,67 @@
 	.text
 main:
 	sw $1, $0
-	sw $at, zero
+	sw $at, $zero
 	sw $2, $0
-	sw $v0, zero
+	sw $v0, $zero
 	sw $3, $0
-	sw $v1, zero
+	sw $v1, $zero
 	sw $4, $0
-	sw $a0, zero
+	sw $a0, $zero
 	sw $5, $0
-	sw $a1, zero
+	sw $a1, $zero
 	sw $6, $0
-	sw $a2, zero
+	sw $a2, $zero
 	sw $7, $0
-	sw $a3, zero
+	sw $a3, $zero
 	sw $8, $0
-	sw $t0, zero
+	sw $t0, $zero
 	sw $9, $0
-	sw $t1, zero
+	sw $t1, $zero
 	sw $10, $0
-	sw $t2, zero
+	sw $t2, $zero
 	sw $11, $0
-	sw $t3, zero
+	sw $t3, $zero
 	sw $12, $0
-	sw $t4, zero
+	sw $t4, $zero
 	sw $13, $0
-	sw $t5, zero
+	sw $t5, $zero
 	sw $14, $0
-	sw $t6, zero
+	sw $t6, $zero
 	sw $15, $0
-	sw $t7, zero
+	sw $t7, $zero
 	sw $16, $0
-	sw $s0, zero
+	sw $s0, $zero
 	sw $17, $0
-	sw $s1, zero
+	sw $s1, $zero
 	sw $18, $0
-	sw $s2, zero
+	sw $s2, $zero
 	sw $19, $0
-	sw $s3, zero
+	sw $s3, $zero
 	sw $20, $0
-	sw $s4, zero
+	sw $s4, $zero
 	sw $21, $0
-	sw $s5, zero
+	sw $s5, $zero
 	sw $22, $0
-	sw $s6, zero
+	sw $s6, $zero
 	sw $23, $0
-	sw $s7, zero
+	sw $s7, $zero
 	sw $24, $0
-	sw $t8, zero
+	sw $t8, $zero
 	sw $25, $0
-	sw $t9, zero
+	sw $t9, $zero
 	sw $26, $0
-	sw $k0, zero
+	sw $k0, $zero
 	sw $27, $0
-	sw $k1, zero
+	sw $k1, $zero
 	sw $28, $0
-	sw $gp, zero
+	sw $gp, $zero
 	sw $29, $0
-	sw $sp, zero
+	sw $sp, $zero
 	sw $30, $0
-	sw $fp, zero
+	sw $fp, $zero
 	sw $31, $0
-	sw $ra, zero
+	sw $ra, $zero
 
 end:
 	j end
diff --git a/tests/pass/win/test07.asm b/tests/pass/win/test07.asm
index 60ad737..f6ac470 100644
--- a/tests/pass/win/test07.asm
+++ b/tests/pass/win/test07.asm
@@ -4,67 +4,67 @@
 	.text
 main:
 	sw $1, $0
-	sw $at, zero
+	sw $at, $zero
 	sw $2, $0
-	sw $v0, zero
+	sw $v0, $zero
 	sw $3, $0
-	sw $v1, zero
+	sw $v1, $zero
 	sw $4, $0
-	sw $a0, zero
+	sw $a0, $zero
 	sw $5, $0
-	sw $a1, zero
+	sw $a1, $zero
 	sw $6, $0
-	sw $a2, zero
+	sw $a2, $zero
 	sw $7, $0
-	sw $a3, zero
+	sw $a3, $zero
 	sw $8, $0
-	sw $t0, zero
+	sw $t0, $zero
 	sw $9, $0
-	sw $t1, zero
+	sw $t1, $zero
 	sw $10, $0
-	sw $t2, zero
+	sw $t2, $zero
 	sw $11, $0
-	sw $t3, zero
+	sw $t3, $zero
 	sw $12, $0
-	sw $t4, zero
+	sw $t4, $zero
 	sw $13, $0
-	sw $t5, zero
+	sw $t5, $zero
 	sw $14, $0
-	sw $t6, zero
+	sw $t6, $zero
 	sw $15, $0
-	sw $t7, zero
+	sw $t7, $zero
 	sw $16, $0
-	sw $s0, zero
+	sw $s0, $zero
 	sw $17, $0
-	sw $s1, zero
+	sw $s1, $zero
 	sw $18, $0
-	sw $s2, zero
+	sw $s2, $zero
 	sw $19, $0
-	sw $s3, zero
+	sw $s3, $zero
 	sw $20, $0
-	sw $s4, zero
+	sw $s4, $zero
 	sw $21, $0
-	sw $s5, zero
+	sw $s5, $zero
 	sw $22, $0
-	sw $s6, zero
+	sw $s6, $zero
 	sw $23, $0
-	sw $s7, zero
+	sw $s7, $zero
 	sw $24, $0
-	sw $t8, zero
+	sw $t8, $zero
 	sw $25, $0
-	sw $t9, zero
+	sw $t9, $zero
 	sw $26, $0
-	sw $k0, zero
+	sw $k0, $zero
 	sw $27, $0
-	sw $k1, zero
+	sw $k1, $zero
 	sw $28, $0
-	sw $gp, zero
+	sw $gp, $zero
 	sw $29, $0
-	sw $sp, zero
+	sw $sp, $zero
 	sw $30, $0
-	sw $fp, zero
+	sw $fp, $zero
 	sw $31, $0
-	sw $ra, zero
+	sw $ra, $zero
 
 end:
 	j end
