diff --git a/CMakeLists.txt b/CMakeLists.txt
index 96423c7..9192628 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,10 +56,19 @@ include_directories(${CMAKE_BINARY_DIR})
 add_executable(unit_tests catch.hpp unit_tests.cpp ${LIB_TEST_SOURCE} ${LIB_SOURCE})
 
 add_executable(virtual_machine_gui_test ${UI_TEST_SOURCE} ${UI_SOURCE} ${LIB_SOURCE})
-target_link_libraries(virtual_machine_gui_test Qt5::Widgets Qt5::Test)
+if(UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCXX)
+  target_link_libraries(virtual_machine_gui_test Qt5::Widgets Qt5::Test pthread)
+else(UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCXX)
+  target_link_libraries(virtual_machine_gui_test Qt5::Widgets Qt5::Test)
+endif()
 
 add_executable(simmips ${SIMMIPS_SOURCE} ${UI_SOURCE} ${LIB_SOURCE})
-target_link_libraries(simmips Qt5::Widgets)
+if(UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCXX)
+  target_link_libraries(simmips Qt5::Widgets pthread)
+else(UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCXX)
+  target_link_libraries(simmips Qt5::Widgets)
+endif()
+
 
 
 # Tests
